CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# dbt_shopify v0.10.0
+- Removes the dependency on [dbt-expectations](https://github.com/calogica/dbt-expectations/releases) and updates [dbt-date](https://github.com/calogica/dbt-date/releases) dependencies to the latest version. ([PR #66](https://github.com/fivetran/dbt_shopify/pull/66/))
```

Comment:
Can we add a subheader here

Reply:
Additionally can we include the a section here that highlights the tests removed

Reply:
updated